internal/adapters/cloudformation/aws/s3/bucket.go

```diff
@@ -27,8 +27,9 @@ func getBuckets(cfFile parser.FileContext) []s3.Bucket {
 				Enabled:   hasVersioning(r),
 				MFADelete: defsecTypes.BoolUnresolvable(r.Metadata()),
 			},
-			Logging: getLogging(r),
-			ACL:     convertAclValue(r.GetStringProperty("AccessControl", "private")),
+			Logging:                 getLogging(r),
+			ACL:                     convertAclValue(r.GetStringProperty("AccessControl", "private")),
+			ObjectLockConfiguration: getObjectLockConfiguration(r, cfFile),
```

Comment:
Think this is resolved by adding `ObjectLockConfiguration` to the `Bucket` definition